_projects/2018/plasmapy/plasma-object.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Plasma class
+desc: Devlop a new Plasma class for loading and storing multiple types of plasma physics data.
+requirements:
+ - Good knowledge of Python, including object-oriented programming
+ - Familiarity with basics of plasma physics
```

Comment:
I wonder if it would be reasonable to make the plasma physics familiarity requirement a bit looser, but saying some background in physics, maybe especially electromagnetism, fluid mechanics, and/or plasma physics.  Most physics undergraduates in the US don't have to take a plasma physics course, but we could provide a chance for the student to learn.

Reply:
Good point, especially as a lot of GSOC students are comp sci rather than physics anyway.